.github/workflows/acceptance-pg14.yml

```
@@ -0,0 +1,124 @@
+        name: Checkout sentry
+
+      - uses: volta-cli/action@v1
+
+      - name: Step configurations
+        id: config
+        run: |
+          echo "::set-output name=yarn-path::$(yarn cache dir)"
+          echo "::set-output name=webpack-path::.webpack_cache"
+          echo "::set-output name=dist-path::src/sentry/static/sentry/dist"
+
+      - name: yarn cache
+        uses: actions/cache@v1 # We are explicitly using v1 due to perf reasons
+        with:
+          path: ${{ steps.config.outputs.yarn-path }}
+          key: ${{ runner.os }}-v2-yarn-${{ hashFiles('yarn.lock', 'api-docs/yarn.lock') }}
+
+      - name: webpack cache
+        uses: actions/cache@v2
+        with:
+          path: ${{ steps.config.outputs.webpack-path }}
+          key: ${{ runner.os }}-v2-webpack-cache-${{ hashFiles('webpack.config.ts') }}
+
+      - name: Install Javascript Dependencies
+        run: |
+          yarn install --frozen-lockfile
+
+      - name: webpack
+        env:
+          WEBPACK_CACHE_PATH: ${{ steps.config.outputs.webpack-path }}
+          SENTRY_INSTRUMENTATION: 1
+          # this is fine to not have for forks, it shouldn't fail
+          SENTRY_WEBPACK_WEBHOOK_SECRET: ${{ secrets.SENTRY_WEBPACK_WEBHOOK_SECRET }}
+        run: |
+          yarn build-acceptance
+
+      # Bundle dist for faster uploading
+      - name: bundle dist
+        run: |
+          tar czf dist.tar.gz ${{ steps.config.outputs.dist-path }}
+
+      - name: Save frontend dist
+        uses: actions/upload-artifact@v2
+        with:
+          retention-days: 3
+          name: frontend-dist-pg14
+          path: dist.tar.gz
+
+  acceptance:
+    name: acceptance-pg14
+    runs-on: ubuntu-20.04
+    timeout-minutes: 20
+    strategy:
+      # This helps not having to run multiple jobs because one fails, thus, reducing resource usage
+      # and reducing the risk that one of many runs would turn red again (read: intermittent tests)
+      fail-fast: false
+      matrix:
+        python-version: [3.8.12]
+        # XXX: When updating this, make sure you also update MATRIX_INSTANCE_TOTAL.
+        instance: [0, 1, 2, 3]
+    env:
+      # XXX: MATRIX_INSTANCE_TOTAL must be hardcoded to the length of strategy.matrix.instance.
+      MATRIX_INSTANCE_TOTAL: 4
+      VISUAL_SNAPSHOT_ENABLE: 1
+      TEST_GROUP_STRATEGY: roundrobin
+
+    steps:
+      - name: Checkout sentry
+        uses: actions/checkout@v2
+
+      - name: Setup sentry env (python ${{ matrix.python-version }})
+        uses: ./.github/actions/setup-sentry
+        id: setup
+        with:
+          python-version: ${{ matrix.python-version }}
+          pip-cache-version: ${{ secrets.PIP_CACHE_VERSION }}
+          snuba: true
+          pg-version: '14'
+
+      - name: Wait for frontend build
+        uses: getsentry/action-wait-for-check@v1.0.0
+        id: wait-for-frontend
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          checkName: create frontend bundle pg14
```

@armenzg @billyvg Would this work?

```suggestion
          checkName: create frontend bundle
```

If so I think acceptance would be faster - we don't need to do a lot of unnecessary webpacking, just wait for frontend build from postgres 9.6 acceptance. setup-sentry takes some time so that helps.

---

It's worth trying.
I think pushing the change to PR would show us if it work.

---

It did not work. That's why I added the pg14 to its name.

https://github.com/getsentry/sentry/runs/5735033883?check_suite_focus=true

---

It doesn't work because they are in different workflows (I think)